### PR TITLE
Update MVC layouts topic

### PR DESCRIPTION
**Pull request by Eilon**

### CODE REVIEW

**Comment by Eilon** [comment]
Oops, build failed, will fix.

**Comment by Rick-Anderson** [comment]
the local build doesn't validate API links, just internal doc/ref links. You can hover over the link in the browser to make sure the link is valid. @danroth27 is updating the API docs tonight. Until he merges that, you can't use new APIs.

**Comment by Rick-Anderson** [comment]
@Eilon  let me know if you'd like me to fix it.

**Comment by Eilon** [comment]
@Rick-Anderson looks like I fixed it, so please review when you have a chance. Thanks!

File: aspnet/mvc/views/layout.rst [hunk 1]
```diff
@@ -82,25 +91,25 @@ A sample ``_ViewImports.cshtml`` file:
 .. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Views/_ViewImports.cshtml
   :language: html
 
-The ``_ViewImports.cshtml`` file for an ASP.NET Core MVC app is typically placed in the ``Views`` folder root. A ``_ViewImports.cshtml`` file can be placed within a controller-associated view folder, in which case it will only be applied to views within that folder. ``_ViewImports`` files are run first at the root level, and then for a ``_ViewImports`` file specified in the controller-associated folder, so settings specified at the root level may be overridden at the folder level.
+The ``_ViewImports.cshtml`` file for an ASP.NET Core MVC app is typically placed in the ``Views`` folder. A ``_ViewImports.cshtml`` file can be placed within any folder, in which case it will only be applied to views within that folder or its subfolders. ``_ViewImports`` files are processed starting at the root level, and then for each folder leading up to the location of the view itself, so settings specified at the root level may be overridden at the folder level.
```
Comment (Rick-Anderson) [inline aspnet/mvc/views/layout.rst]: can we say **and** its subfolders in lieu of or?

**Comment by Rick-Anderson** [comment]
Wow - really nice improvements. :shipit:

File: aspnet/mvc/views/layout.rst (hunk 1 above)
Comment (Eilon) [inline aspnet/mvc/views/layout.rst]: Ah sure.

Comment (Eilon) [inline aspnet/mvc/views/layout.rst]: Done.

**Pull request merged**